ibm/mas_devops/roles/suite_upgrade/defaults/main.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+mas_catalog_source: "{{ lookup('env', 'MAS_CATALOG_SOURCE') | default('ibm-operator-catalog', true) }}"
+mas_instance_id: "{{ lookup('env', 'MAS_INSTANCE_ID') }}"
+mas_namespace: "mas-{{ mas_instance_id }}-core"
+
+mas_upgrade_supported_channels: ['maint86xdev', '8.6.x']
```

Comment:
remove dev channel?

Reply:
yeah, i think at this point we can remove it safely.